enterprise/kernel/src/main/java/org/neo4j/kernel/monitoring/tracing/VerboseTracerFactory.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.monitoring.tracing;
+
+import org.neo4j.helpers.Service;
+import org.neo4j.io.pagecache.tracing.PageCacheTracer;
+import org.neo4j.kernel.impl.util.JobScheduler;
+import org.neo4j.kernel.monitoring.Monitors;
+import org.neo4j.logging.Log;
+import org.neo4j.time.SystemNanoClock;
+
+@Service.Implementation( TracerFactory.class )
+public class VerboseTracerFactory extends DefaultTracerFactory
```

Comment:
This won't be available unless you add it to a META-INF/services file.

Reply:
unless you rely on ServiceProcessor or any other auto generator to do that automatically:) he

Reply:
Are you sure that's still plugged in and running?

Reply:
yep, will gonna add integration test that uses verbose tracer